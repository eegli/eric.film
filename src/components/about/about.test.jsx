import { shallow } from '@/src/tests/testUtils';
import About from './about.component';

describe('About', () => {
  it('renders shallow wrapper', () => {
    const shallowWrapper = shallow(<About />);
    expect(shallowWrapper).toMatchSnapshot();
  });
});
