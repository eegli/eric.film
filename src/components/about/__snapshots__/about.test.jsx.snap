// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders shallow wrapper 1`] = `
<Fragment>
  <aboutstyles__AboutContainer>
    <CardComponent />
    <hr
      style={
        Object {
          "width": "80%",
        }
      }
    />
    <ResumeComponent />
  </aboutstyles__AboutContainer>
</Fragment>
`;
